test/test_helper.rb

```diff
@@ -23,10 +23,13 @@
 ActiveSupport::TestCase.test_order = :sorted
 
 class LoggerStub
-  include ContextualLogger
-  attr_accessor :logged
+  include ContextualLogger::LoggerMixin
+  attr_accessor :logged, :level
 
   def initialize
+    @level = Logger::Severity::DEBUG
+    @progname = nil
+    @logdev = nil
```

Comment:
`level` is now needed by the `LoggerMixin`. And the other two generated `ruby -w` warnings.